style.css

```diff
@@ -9,11 +9,11 @@ body {
   margin: 1rem;
   padding: 1rem;
   width: 327px;
+  font-family: 'Inter', sans-serif;
 }
```

Comment:
- [ ] [OPTIONAL] It looks like there is a problem with your project that needs a small fix. You can see in the picture that there is more space on the right side of the project than on the left side, which is not good. please try to fix this!

![image](https://user-images.githubusercontent.com/63915024/185664029-94dffc82-0252-4c8a-b83b-3032c8d583fe.png)